ruma-identity-service-api/src/association/email/create_email_validation_session/v2.rs

```diff
@@ -0,0 +1,56 @@
+//! [POST /_matrix/identity/v2/validate/email/requestToken](https://matrix.org/docs/spec/identity_service/r0.3.0#post-matrix-identity-v2-validate-email-requesttoken)
+
+use ruma_api::ruma_api;
+
+ruma_api! {
+    metadata: {
+        description: "Create a session for validating an email address.",
+        method: POST,
+        name: "create_email_validation_session",
+        path: "/_matrix/identity/v2/validate/email/requestToken",
+        authentication: AccessToken,
+        rate_limited: false,
+    }
+
+    request: {
+        /// A unique string generated by the client, and used to identify the validation attempt.
+        pub client_secret: &'a str,
+
+        /// The email address to validate.
+        pub email: &'a str,
+
+        /// The server will only send an email if the send_attempt is a number greater than the
+        /// most recent one which it has seen, scoped to that email + client_secret pair.
+        pub send_attempt: js_int::UInt,
```

Comment:
For consistency, please import `js_int::UInt` at the top of the file and refer to it as just `UInt` here and below.